src/services/scheduler.ts

```diff
@@ -146,6 +146,76 @@ export async function start(dbId: string, dbUser: string) {
           }
         }
       },
+      rpc: async (payload: any) => {
+        const { params, opid, modulename, methodname } = payload;
+        let output;
+        let error;
+        const user = await MetadataRepo.getUserFromDbId(dbId);
+        const uid = user?.id;
+        const email = user?.email;
+        const dbAlias = user?.iasqlDatabases?.[0]?.alias;
+        try {
+          const versionString = await TypeormWrapper.getVersionString(dbId);
+          const Modules = (modules as any)[versionString];
+          if (!Modules[modulename]) throw new Error(`Module ${modulename} not found`);
+          if (!Modules[modulename][methodname]) {
+            throw new Error(`Method ${methodname} in module ${modulename} not found`);
+          }
+          output = await Modules[modulename][methodname](params);
+          // once the rpc completes updating the `end_date`
+          // will complete the polling
+          const query = `
+            update iasql_rpc
+            set end_date = now(), output = '${output}'
+            where opid = uuid('${opid}');
+          `;
+          logger.debug(query);
```

Comment:
is it a leftover or you want to keep it for testing purposes?

Reply:
So the structure is similar to the current `operation` and that debug log was there. I think it would be easier to debug in the case is needed having that log there.

Reply:
Thinking about it, I've removed it. that query could be seen in the postgres container logs.